contracts/EverlongStrategy.sol

```diff
@@ -121,9 +122,19 @@ contract EverlongStrategy is BaseStrategy {
     ///         If false, use the Hyperdrive's `vaultSharesToken`.
     bool public immutable asBase;
 
+    /// @notice Whether the strategy asset is a wrapped hyperdrive token.
```

Comment:
"wrapped hyperdrive token" doesn't really make sense to me. wstETH isn't a wrapped Hyperdrive token, it's a wrapped variant of stETH.

Reply:
Can see why this language is a bit confusing... intent here was that "hyperdrive token" means hyperdrive's base token or vault shares token consistent with the `hyperdriveToken` variable